Utilities/bootstrap

```diff
@@ -232,7 +232,14 @@ def get_build_target(args):
     if platform.system() == 'Darwin':
         return "x86_64-apple-macosx"
     elif platform.system() == 'Linux':
-        if platform.machine() == 'x86_64':
+        if 'ANDROID_DATA' in os.environ:
```

Comment:
Hey, wanna try using the new `swift -print-target-info` to get the triple automatically?

Reply:
I'll try it out.